src/vidible-player.js

```diff
@@ -206,7 +220,13 @@
                             destroyPlayer();
 
                             // Create new Vidible element
-                            vidibleQueueLoader.queueForProcessing(videoId, element, initPlayer);
+                            vidibleQueueLoader.queueForProcessing(
+                                {
+                                    videoId:videoId,
+                                    autoplay:scope.autoplay
```

Comment:
Let's keep this method call simple - queueForProcessing(videoId, isAutoplay, element, initPlayer)
and take care for what to put it the queue in the implementation of queueForProcessing